test/coercive.rb

```diff
@@ -98,6 +99,14 @@ def assert_coercion_error(errors)
         assert_coercion_error(expected_errors) { @coercion.call("foo" => bad) }
       end
     end
+
+    it "tries to reproduce a horrible bug" do
+      attributes = { "bar" => false }
+
+      expected = { "bar" => false }
+
+      assert_equal expected, @coercion.call(attributes)
+    end
```

Comment:
This didn't turn out to be what the bug really was, right?

Reply:
lol! no, it wasn't. It was having two `attribute :foo`. I'll delete this